StripeUICore/StripeUICore.xcodeproj/project.pbxproj

```diff
@@ -91,6 +91,7 @@
 		B1B177526E04EE65A9D1C64A /* DynamicHeightContainerView.swift in Sources */ = {isa = PBXBuildFile; fileRef = B9E7CFED5747279A5976D7BA /* DynamicHeightContainerView.swift */; };
 		B6107E5E2D6E116417D22DD9 /* StripeUICoreBundleLocator.swift in Sources */ = {isa = PBXBuildFile; fileRef = E68363D1C6BB6468AC1DDA4C /* StripeUICoreBundleLocator.swift */; };
 		B65F5B032E60C5EB001425CC /* LiquidGlassHelpers.swift in Sources */ = {isa = PBXBuildFile; fileRef = B65F5B022E60C5EB001425CC /* LiquidGlassHelpers.swift */; };
+		B6F905562E7E0BC800942BA5 /* ElementsAppearance.swift in Sources */ = {isa = PBXBuildFile; fileRef = B6F905552E7E0BC500942BA5 /* ElementsAppearance.swift */; };
```

Comment:
You can remove these changes, the xcodeproj will now pull in source files automatically